container/lxd/image_test.go

```diff
@@ -27,10 +27,34 @@ func (t *imageSuite) patch(svr lxdclient.ImageServer) {
 	lxd.PatchConnectRemote(t, svr)
 }
 
+func (s *imageSuite) TestCopyImageUsesPassedCallback(c *gc.C) {
```

Comment:
I don't see how this is testing that the passed callback is actually getting invoked.
Since you're passing a 'noopCallback' and don't seem to be checking if it would have ever been called.
Is it just that passing in the object doesn't cause a failure?

Reply:
The non-nil callback value causes _AddHandler_ to be called. It isn't otherwise.

The callback itself is wrapped in a method that looks for specific keys in the Operation's _Metadata_ member. Since there are no outstanding calls that we have not EXPECTed, I can't yet see how to force the invocation.